### ISSUE TITLE
False negatives for vendor-prefixed values in value lists in function-linear-gradient-no-nonstandard-direction

### ISSUE THREAD

**Issue by pfructuoso** [stylelint/stylelint#2481]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

It´s a bug. [False positives and negatives when vendor-prefixed values are used in function-linear-gradient-no-nonstandard-direction](https://github.com/stylelint/stylelint/issues/1939) is still happening when using multiple values for `background-image`

> Which rule, if any, is this issue related to?

`function-linear-gradient-no-nonstandard-direction`

 > What CSS is needed to reproduce this issue?

```css
   .node-preview-backlink {
     /* false negative */
     background: url(image.png), -webkit-linear-gradient(top, #012345, #ABCDEF);

     /* Working as expected */
     background: -webkit-linear-gradient(top, #012345, #ABCDEF), url(image.png);
   }
```

> What stylelint configuration is needed to reproduce this issue?

```json
   {
"rules": {
       "function-linear-gradient-no-nonstandard-direction": true
       }
}
```

> Which version of stylelint are you using?

`7.10.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI interface.
`stylelint test.css`

 > Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No, it´s standard syntax

> What did you expect to happen?

Both cases on the example have the same code so no warnings should be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

In "false negative" the following warning wa flagged:

```shell
   test.css
3:55  ✖  Unexpected nonstandard direction   function-linear-gradient-no-nonstandard-direction
```

I tried to figure out what is happening by myself and it looks like in
´lib/rules/function-linear-gradient-no-nonstandard-direction/index.js´

**Comment by jeddy3** [comment]
@pfructuoso Thank you for the report and PR!

**Comment by pfructuoso** [comment]
Can someone with better git-foo than I advise on how best to resolve the problem with the PR #2482? Thanks.

**Comment by davidtheclark** [comment]
Closed by #2496.
